CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Aries Cloud Agent Python Changelog
 
+## 0.12.3rc0
+
+### December 16, 2024
+
+A patch release to add address a bug found in the Linked Data Verifiable Credential handling for multi-tenant holders. The bug was fixed in the main branch, [PR 3391 - BREAKING: VCHolder multitenant binding](https://github.com/openwallet-foundation/acapy/pull/3391), and with this release is backported to 0.12 Long Term Support branch. Prior to this release, holder credentials received into a single tenant wallet were visible in the admin wallet, and when used in "admin insecure mode", visibile in all tenants.
```

Comment:
Maybe just `Prior to this release, holders credentials received into a tenant wallet were actually received into the multitenant admin wallet."

Reply:
Nice — fixed.